spec/index.bs

```diff
@@ -795,6 +800,11 @@ interface GPUDevice {
     GPUCommandEncoder createCommandEncoder(GPUCommandEncoderDescriptor descriptor);
 
     GPUQueue getQueue();
+
+    // Calling configureSwapChain a second time invalidates the previous one,
+    // and all of the textures it’s produced.
+    GPUSwapChain configureSwapChain(GPUSwapChainDescriptor descriptor);
```

Comment:
todo: do configureSwapChain and getSwapChainPreferredFormat need to know what canvas or ImageBitmapRenderingContext they're targeting? likely yes